netstandard/ref/System.Security.cs

```diff
@@ -48,7 +48,6 @@ public sealed partial class SecureString : System.IDisposable
         [System.Runtime.ExceptionServices.HandleProcessCorruptedStateExceptionsAttribute]
         public void SetAt(int index, char c) { }
     }
-    [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
     public static partial class SecureStringMarshal
     {
```

Comment:
Presumably we want to keep hiding this one as the type was a .NET Core 1.x helper that is now replaced with the actual instance method.

Reply:
This one will not be replaced with instance methods as they are all static helpers. The methods however have been added back to the Marshal type so there isn't any real need for them but it also doesn't hurt anything.

Reply:
Alright, I'll keep hiding this type then as it's just duplication.